TS3Hook/dllmain.cpp

```diff
@@ -98,9 +100,20 @@ void create_config(const LPCWSTR file_name)
 	WritePrivateProfileString(lpSection, L"ignorecmds", L"", file_name);
 	WritePrivateProfileString(lpSection, L"blockcmds", L"", file_name);
 	WritePrivateProfileString(lpSection, L"injectcmd", L" msg=~cmd", file_name);
+	WritePrivateProfileString(lpSection, L"clientversion", L"", file_name);
```

Comment:
Please add a default value here

Reply:
Forget it. don't add a default value as i want this to be disabled by default as said in https://github.com/ReSpeak/TS3Hook/pull/18#discussion_r161377982

Just add a comment to the code how the config line should be formatted

`version|platform|versionsign`?

Reply:
Will take a look at this in a moment

Reply:
Dunno how to do that

Reply:
What is your code replacing with what then? just give me a quick list of 
- change = changed

Reply:
I don't understand what you want from me :D

Reply:
What does your pull request do. Thats what i want (to know)

Reply:
It adds possibility to change (fake) client platform which can be set in HookConf.ini file